src/main/java/com/wultra/security/userdatastore/service/UserClaimsService.java

```diff
@@ -73,6 +73,7 @@ public Object fetchUserClaims(final String userId) {
         }
     }
 
+    @Transactional
```

Comment:
Ok, no problem, it is only a matter of style. I added the annotation to the class level to avoid many declarations, but the truth is, that this is probably more straightforward.